### ISSUE TITLE
Discuss: Potential Chapter table updates

### ISSUE THREAD

**Issue by viviancan** [Iridescent-CM/technovation-app#4506]

**Comment by viviancan** [comment]
Based on #4446 we will need a "seasons" filter. Should we add a seasons field to the Chapter table?

Some other ideas 
- created at/updated at
- created by (like which admin created this chapter?)

**Comment by viviancan** [comment]
@shaun-technovation and @dboyer

**Comment by dboyer** [comment]
Definitely a seasons filter. We might want to run the created at/updated at and created by past PT to see if they would use them or if they have other ideas.

**Comment by dboyer** [comment]
@shaun-technovation Good catch on this. To confirm, would this both allow us to see which chapters/clubs were active in which seasons as well as which chapter/club a student or mentor belonged to in a season? Or are those two separate tasks?
